src/dreyfus_httpd.erl

```diff
@@ -17,6 +17,9 @@
 
 -export([handle_search_req/3, handle_info_req/3, handle_disk_size_req/3,
          handle_cleanup_req/2, handle_analyze_req/1]).
+
+-export([parse_index_params/1]).
```

Comment:
shouldn't this be -export([parse_index_params/2]).

Reply:
I've removed it. It was added on an old commit and never used.